test/remote/gateways/remote_sum_up_test.rb

```diff
@@ -51,6 +51,16 @@ def test_successful_purchase_with_more_options
     assert_equal 'PAID', response.message
   end
 
+  def test_successful_purchase_with_partner_id
+    options = {
+      partner_id: 'PartnerId',
+      order_id: SecureRandom.uuid
+    }
+
+    response = @gateway.purchase(@amount, @credit_card, options)
```

Comment:
Since this is a remote test, could be good idea add a ` assert_success response` 💡

Reply:
Ah yes, Gustavo pointed this out as well. I opted to leave it out since the purchase transactions are failing.